tests/ui/compiletest-self-test/show-std-source-without-flag.rs

```diff
@@ -0,0 +1,22 @@
+// This test without //@ show-std-source directive, as a contrast to show-std-source.rs
```

Comment:
Please combine these two tests into one by use of revisions (https://rustc-dev-guide.rust-lang.org/tests/compiletest.html#revisions).

E.g.,

```suggestion
//@ revisions: show hide
//@[show] show-std-source
```